test/common.mak

```diff
@@ -14,11 +14,13 @@ SRC:=src
 GENERATED:=./generated
 ROOT:=$(GENERATED)/$(OS)/$(BUILD)/$(MODEL)
 
+OPTIONAL_PIC:=$(if $(PIC),-fPIC,)
```

Comment:
@jmdavis why not pass `OPTIONAL_PIC` (as @MartinNowak suggested), instead of `PIC`, so the `OPTIONAL_PIC` has to be calculated again?